image.yaml

```diff
@@ -20,10 +20,24 @@ envs:
       value: "7.3.0.CD04"
     - name: "PRODUCT_VERSION"
       value: "7.3.0.CD04"
+packages:
+    repositories:
+        - jboss-os
 modules:
       repositories:
           - path: modules
+          - name: cct_module
+            git:
+                url: https://github.com/jboss-openshift/cct_module.git
+                ref: master
+          - name: jboss-eap-7-image
+            git:
+                url: https://github.com/jboss-container-images/jboss-eap-7-image.git
+                ref: eap-cd-dev
```

Comment:
@slaskawi I would probably suggest to use something more stable than `eap-cd-dev` branch. Maybe [CD15](https://github.com/jboss-container-images/jboss-eap-7-image/tree/CD15) instead of it? (just for the case some breaking change is accepted into `eap-cd-dev` branch, the RH-SSO CD image still uses stable last EAP CD tag)

But in general (from technical PoV) the PR LGTM / works.

Reply:
Implemented as we agreed. Thanks a lot for suggestion!